src/components/Login/index.jsx

```diff
@@ -4,10 +4,14 @@ import PropTypes from "prop-types";
 import { Button } from "@sendgrid/ui-components/button";
 import { Row } from "../Row";
 import { Column } from "../Column";
+import { WriteSelectors } from "./selectors";
 import "./index.scss";
 
 const InvalidCredentialsAlert = ({ handleAlertClose }) => (
-  <div data-test="invalid-credentials-alert" className="alert alert-danger">
+  <div
+    {...WriteSelectors.invalidCredentialsAlert}
```

Comment:
Spreading out the write selectors like { data-test: "invalid-credentials-alert } onto the component's DOM elements so we can refer to them in tests